pennylane/drawer/mpldrawer.py

```diff
@@ -884,6 +889,55 @@ def measure(self, layer, wires, text=None, box_options=None, lines_options=None)
                 layer + 0.05 * self._box_length, wires + 0.225, text, fontsize=(self.fontsize - 2)
             )
 
+    def _y(self, wire):
+        """Convert a wire number to a y coordinate. Wire numbers greater than the number of
+        quantum wires are treated as classical wires."""
+        if wire < self.n_wires:
+            return wire
+        return self.n_wires + self._cwire_scaling * (wire - self.n_wires) - 0.4
+
+    def classical_wire(self, layers, wires) -> None:
+        """Draw a classical control line.
+
+        Args:
+            layers: a list of x coordinates for the classical wire
+            wires: a list of y coordinates for the classical wire. Wire numbers
+                greater than the number of quantum wires will be scaled as classical wires.
+
+        """
+        s0 = path_effects.Stroke(
+            linewidth=5 * plt.rcParams["lines.linewidth"], foreground=plt.rcParams["lines.color"]
+        )
+
+        s1 = path_effects.Stroke(
+            linewidth=3 * plt.rcParams["lines.linewidth"],
+            foreground=plt.rcParams["figure.facecolor"],
+        )
+
+        line = plt.Line2D(layers, [self._y(w) for w in wires], path_effects=[s0, s1], zorder=1)
+        self.ax.add_line(line)
+
+    def cwire_join(self, layer, wire, erase_right=False):
+        """Erase the horizontal edges of an intersection between classical wires. By default, erases
+        only the left edge.
+
+        Args:
+            layer: the x corrdinate for the classical wire intersection
+            wire: the classical wire coordinate for the intersection
+            erase_right=False(bool):  whether or not to erase the right side of the intersection
+                in addition to the left.
+
+        """
+        xs = (layer - 0.2, layer + 0.2) if erase_right else (layer - 0.2, layer)
+        line = plt.Line2D(
+            xs,
+            (self._y(wire), self._y(wire)),
+            zorder=2,
+            color=plt.rcParams["figure.facecolor"],
```

Comment:
If you use `lines.color` it'll use the same color as the line and the drawing will look the same. The intention is to overlap the intersection with a "white" line so that you're effectively erasing it.

Reply:
ohhh this is the eraser 😅 ignore me